contrib/platform/src/com/sun/jna/platform/win32/Kernel32Util.java

```diff
@@ -509,53 +470,6 @@ public static long findEnvironmentStringBlockEntryEnd(Pointer lpszEnvironmentBlo
         }
     }
 
-    /**
-     * <P>Attempts to determine whether the data block uses {@code wchar_t}
-     * instead of &quot;plain old&quot; {@code char}s. It does that by reading
-     * 2 bytes from the specified offset - the character value and its charset
-     * indicator - and examining them as follows:</P>
-     * <UL>
-     *      <LI>
-     *      If the charset indicator is non-zero then it is assumed to be
-     *      a &quot;plain old&quot; {@code char}s data block. <B>Note:</B>
-     *      the assumption is that the environment variable <U>name</U> (at
-     *      least) is ASCII.
-     *      </LI>
-     *
-     *      <LI>
-     *      Otherwise (i.e., zero charset indicator), it is assumed to be
-     *      a {@code wchar_t}
-     *      </LI>
-     * </UL>
-     * <B>Note:</B> the code takes into account the {@link ByteOrder} even though
-     * only {@link ByteOrder#LITTLE_ENDIAN} is the likely one
-     * @param lpszEnvironmentBlock The environment block as received from the
-     * <A HREF="https://msdn.microsoft.com/en-us/library/windows/desktop/ms683187(v=vs.85).aspx">GetEnvironmentStrings</A>
-     * function
-     * @param offset offset
-     * @return {@code true} if the block contains {@code wchar_t} instead of
-     * &quot;plain old&quot; {@code char}s
-     */
-    public static boolean isWideCharEnvironmentStringBlock(Pointer lpszEnvironmentBlock, long offset) {
```

Comment:
Where did this whole method go?

Reply:
Was intended to mark as deprecated, but somehow got removed during my squashing. Will be restored on next push.